### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by dawnwages** [djangonaut-space/wagtail-indymeet#4]
WCAG AA level Accessibility compliance
https://wagtail.org/accessibility/

- Accessibility checker: https://www.siteimprove.com/glossary/wcag-compliance-checker/
- Accessibility checker: https://www.accessibilitychecker.org/

**Comment by dawnwages** [comment]
Reach out to Wagtail's Accessibility team + Thibaud

**Comment by sabderemane** [comment]
Note:
- Some ALT missing
- Skip to content button missing
